cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AliasesTest.scala

```diff
@@ -0,0 +1,304 @@
+            |
+            |* The number of columns returned has been reduced with the `YIELD` clause.
+            |* The order of the returned columns has been changed.
+            |* The results have been filtered to only show database alias names containing 'e'.
+            |* The results are ordered by the 'database' column using `ORDER BY`.
+            |
+            |It is also possible to use `SKIP` and `LIMIT` to paginate the results.
+            |""".stripMargin)
+        resultTable()
+      }
+    }
+    section(title="Creating database aliases", id="alias-management-create-database-alias", role = "enterprise-edition") {
+      p("Aliases can be created using `CREATE ALIAS`.")
+      p("include::alias-management-syntax-create-alias.asciidoc[]")
+      p(
+        """This command is optionally idempotent, with the default behavior to fail with an error if the database alias already exists.
+          |Inserting `IF NOT EXISTS` after the alias name ensures that no error is returned and nothing happens should a database alias with that name already exist.
+          |Adding `OR REPLACE` to the command will result in any existing database alias being deleted and a new one created.
+          |`CREATE OR REPLACE ALIAS` will fail if there is an existing database with the same name.""".stripMargin)
+      note {
+        p("The `IF NOT EXISTS` and `OR REPLACE` parts of this command cannot be used together.")
+      }
+      note {
+        p("""Alias names are subject to the <<cypher-manual#cypher-naming, standard Cypher restrictions on valid identifiers>>.
+            |The following naming rules apply:""")
+        p(
+          """
+            |* A name is a valid identifier, additionally allowing dots e.g. `main.alias`.
+            |* Name length can be up to 65534 characters.
+            |* Names cannot end with dots.
+            |* Names that begin with an underscore or with the prefix `system` are reserved for internal use.
+            |* Non-alphabetic characters, including numbers, symbols and whitespace characters, can be used in names, but must be escaped using backticks.
+            |""".stripMargin)
+      }
+
+      section(title="Creating local database aliases", id="database-management-create-local-database-alias", role = "enterprise-edition") {
+        p("Local aliases are created with a target database.")
+        query("CREATE ALIAS `northwind` FOR DATABASE `northwind-graph-2021`", ResultAssertions(r => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        p("When a local database alias has been created, it will show up in the aliases column provided by the command `SHOW DATABASES` and in the `SHOW ALIAS FOR DATABASE` command.")
+        query("SHOW DATABASE `northwind`", assertNameField("northwind-graph-2021")) {
+          resultTable()
+        }
+        query("SHOW ALIASES FOR DATABASE", assertAliasesShown) {
+          resultTable()
+        }
```

Comment:
It might be a bit confusing that this is listing four different alias including a remote one, for someone who just are reading this section. Maybe we should add a YIELD and only print out the one that was just created (like we do for remote)?